docs/source/migrate_experiment_tracking.md

```diff
@@ -0,0 +1,141 @@
+
+- The project root 
+- The `.viz` folder within the project root
+- The `data` folder
+
+
+## Update Dataset Configurations in Catalog
+
+Update the dataset configurations in your `catalog.yml` to transition to `kedro-mlflow`. Refer to the table below for the changes:
+
+### Dataset migration Table
+
+| Kedro-Viz Dataset Type         | MLflow Dataset Type        | Update Instructions                                      |
+|---------------------------------|----------------------------|---------------------------------------------------------|
+| `tracking.MetricsDataset`      | `MlflowMetricDataset`      | Update type to `MlflowMetricDataset`.                  |
+| `tracking.JSONDataset`         | `MlflowArtifactDataset`    | Wrap within `MlflowArtifactDataset` as `json.JSONDataset`. |
+| `plotly.plotlyDataset`         | `MlflowArtifactDataset`    | Wrap within `MlflowArtifactDataset` as `plotly.HTMLDataset`. |
+| `plotly.JSONDataset`           | `MlflowArtifactDataset`    | Wrap within `MlflowArtifactDataset` as `plotly.HTMLDataset`. |
+| `matplotlib.MatplotlibWriter`  | `MlflowArtifactDataset`    | Wrap within `MlflowArtifactDataset`.                   |
+
+### Metrics Dataset
+For `tracking.MetricsDataset`, update its type to `MlflowMetricDataset`:
+
+Before:
+```yaml
+metrics:
+  type: tracking.MetricsDataset
+  filepath: data/09_tracking/metrics.json
+  versioned: true
+```
+
+After:
+```yaml
+metrics:
+  type: kedro_mlflow.io.metrics.MlflowMetricDataset
+```
+
+### JSON Dataset
+For `tracking.JSONDataset`, wrap it within `MlflowArtifactDataset` and configure it as `json.JSONDataset`:
+
+Before:
+```yaml
+companies_columns:
+  type: tracking.JSONDataset
+  filepath: data/09_tracking/json_data.json
+  versioned: true
+```
+
+After:
+```yaml
+companies_columns:
+  type: kedro_mlflow.io.artifacts.MlflowArtifactDataset
+  dataset:
+    type: json.JSONDataset
+    filepath: data/02_intermediate/companies_columns.json
+```
+
+### Plotly Dataset
+For `plotly.plotlyDataset` and `plotly.JSONDataset`, wrap it within `MlflowArtifactDataset` and configure it as `plotly.HTMLDataset` to render interactive plots in the MLflow UI:
+
+Before:
+```yaml
+plotly_json_data:
+  type: plotly.JSONDataset
+  filepath: data/09_tracking/plotly.json
+```
+
+After:
+```yaml
+plotly_json_data:
+  type: kedro_mlflow.io.artifacts.MlflowArtifactDataset
+  dataset:
+    type: plotly.HTMLDataset
+    filepath: data/08_reporting/plotly.html
+```
+
+### Matplotlib Writer
+For `matplotlib.MatplotlibWriter`, wrap it within `MlflowArtifactDataset`:
+
+Before:
+```yaml
+confusion_matrix:
+  type: matplotlib.MatplotlibWriter
+  filepath: data/09_tracking/confusion_matrix.png
+  versioned: true
+```
+
+After:
+```yaml
+confusion_matrix:
+  type: kedro_mlflow.io.artifacts.MlflowArtifactDataset
+  dataset:
+    type: matplotlib.MatplotlibWriter
+    filepath: data/07_reporting/confusion_matrix.png
```

Comment:
Nit: the after filepaths seem a bit random with regards to the data engineering layer conventions. maybe just pick the same for all?